packages/langchain/README.md

```diff
@@ -103,6 +97,27 @@ const embeddingClient = new AzureOpenAiEmbeddingClient({
 });
 ```
 
+#### Custom Destination
+
+When initializing the `AzureOpenAiChatClient` and `AzureOpenAiEmbeddingClient` clients, it is possible to provide a custom destination.
+For example, when targeting a destination with the name `my-destination`, the following code can be used:
+
+```ts
+const chatClient = new AzureOpenAiChatClient(
+  {
+    modelName: 'gpt-4o',
+    modelVersion: '24-07-2021',
+    resourceGroup: 'my-resource-group'
+  },
+  {
+    destinationName: 'my-destination'
+  }
+);
+```
```

Comment:
[q] Purely a design question, maybe I am overlooking something.
What was the reasoning for accepting the destination as part of the client initialisation as opposed to accepting it for the `run` method instead like in the AI Core clients?

Right now, it is not possible to use a set of client configurations with multiple destinations and they are tightly coupled.

Reply:
This is purely a design consideration. I can hardly think of a case where a user would reuse a single client with the same, e.g., Azure OpenAI or orchestration configuration (meaning same `resourceGroup`, same model and everything) for different destinations (meaning different subaccounts with different AI Core instances). The run methods on the other hand, is generated and we don't have a client for it, so the best we can do.

Reply:
Hmm, I was thinking about using the same client for destinations targeting different landscapes where the AI Core service is deployed.
But, you are right, we already couple `resourceGroup` into the configuration now, which is tied to a tenant or subaccount.

Reply:
I mean if they want the same configuration to be used, just store configuration in a separate variable, and initialize a new client. Pretty straightforward.

Reply:
Also, I don't think `run` is generated, it is handwritten for these clients.